src/main/java/io/lumigo/core/utils/StringUtils.java

```diff
@@ -54,4 +59,24 @@ public static String extractStringForStream(InputStream inputStream, int size) {
         }
         return null;
     }
+
+    private static String bytesToHex(byte[] bytes) {
+        StringBuilder sb = new StringBuilder();
+        for (byte b : bytes) {
+            sb.append(String.format("%02x", b));
+        }
+        return sb.toString();
+    }
+
+    public static String buildMd5Hash(Map<String, AttributeValue> item) {
```

Comment:
I think the API should not get AttributeValue which is AWS property.
Looks like we need 2 modify it to string outside of this function

Reply:
The api will be a string to md5